extract-metadata.py

```diff
@@ -0,0 +1,71 @@
+from dandi.dandiapi import DandiAPIClient
+from dandi.dandiapi import RemoteDandiset
+import numpy as np
+import pandas as pd
+from pathlib import Path
+
+client = DandiAPIClient("https://api.lincbrain.org/api")
+client.dandi_authenticate()
+
+# Create dataframe of all assets across all Dandisets
+dandisets = []
+for data in client.paginate("/dandisets/"):
+    dandisets.append(RemoteDandiset.from_data(client, data))
+print(f"Number of Dandisets: {len(dandisets)}")
```

Comment:
This code is redundant, since you already iterate over the dandisets in https://github.com/lincbrain/linc-dashboard/pull/4/files#r2058600326